invisible_cities/cities/base_cities.py

```diff
@@ -763,15 +763,16 @@ def make_tracks(self, evt_number : float , evt_time: float,
         track_graphs = paf.make_track_graphs(voxels, self.voxel_dimensions) # type: Sequence[Graph]
 
         for trk in track_graphs:
-            distances = paf.shortest_paths(track_graph)
-            a,b       = paf.find_extrema(distances) # type: Sequence[Voxel]
-            Ea = energy_within_radius(distances[a], self.blob_radius) # type: float
-            Eb = energy_within_radius(distances[b], self.blob_radius)
-            blob_a = Blob(a, Ea) # type: Blob
-            blob_b = Blob(b, Eb)
-            blobs = (blob_a, blob_b) if Ea < Eb else (blob_b, blob_a) # type: Tuple[Blob, Blob]
+            distances = paf.shortest_paths(trk)
+            a,b       = paf.find_extrema(distances) # type: Tuple[Voxel, Voxel]
```

Comment:
Why use comments to specify the type, rather than Python's annotation syntax, if 

+ you advocate the use of Python's built-in annotation syntax
+ it is available in all the versions of Python we support!